src/entities/trades/curve/types.ts

```diff
@@ -36,14 +37,16 @@ export interface CurveTradeQuote {
   contract: Contract
 }
 
-export interface CurveTradeBestTradeExactInParams {
+interface CurveGetTradeCommonParams {
+  maximumSlippage: Percent
+  receiver?: string
+}
+export interface CurveTradeBestTradeExactInParams extends CurveGetTradeCommonParams {
```

Comment:
There are lines separating other objects. It's not functionally important but let's lint the code.

Reply:
Addressed in https://github.com/levelkdev/swapr-sdk/pull/80/commits/55595d108ce509867f961b739bfea556f9cc792f